packages/msw-addon/src/mswDecorator.js

```diff
@@ -1,23 +1,26 @@
-import { makeDecorator } from '@storybook/addons';
-import { setupWorker } from 'msw';
+import { makeDecorator } from '@storybook/addons'
+import { setupWorker } from 'msw'
 
-let worker;
+const IS_BROWSER = typeof global.process === 'undefined'
 
-export function initializeWorker(options) {
-  if (typeof global.process === 'undefined') {
-    worker = setupWorker();
-    worker.start(options);
+export let worker
+
+export const mswDecorator = (options) => {
+  if (IS_BROWSER) {
+    worker = setupWorker()
+    worker.start(options)
   }
-}
 
-export const mswDecorator = makeDecorator({
-  name: 'withMsw',
-  parameterName: 'msw',
-  wrapper: (storyFn, context, { parameters = [] }) => {
```

Comment:
I wonder if `parameters` is a special property here and whether we can rename it to `handlers`.

Reply:
Yeah, its a special property. This line makes the mocking not work on local.

You might have just checked that cards are showing on Storybook but that happens because the API is then going to Star Wars API hosted server.

This should have been reported by the Chromatic Integration. On their dashboard I can see the breaking change but its not reported due to repo transfer. let me check how to fix that.

https://www.chromatic.com/test?appId=5fe732c7dcf9270021051ab0&id=60547bb44297380021ce4592

<img width="1401" alt="image" src="https://user-images.githubusercontent.com/15871340/112724788-fd91d480-8f3a-11eb-8521-46e320a285da.png">

Reply:
Thanks for debugging that. I've enabled Chromatic and Netlify, so the build should be properly reported now.